services/gtfs-rt-archiver-v3/gtfs_rt_archiver_v3/ticker.py

```diff
@@ -90,6 +98,8 @@ def tick(second: int) -> None:
     schedule.every().minute.at(":00").do(tick, second=0)
     schedule.every().minute.at(":20").do(tick, second=20)
     schedule.every().minute.at(":40").do(tick, second=40)
+    schedule.every().minute.at(":45").do(get_configs)
```

Comment:
This syntax makes it look like it would happen every minute (as opposed to every 5 minutes) is that wrong?

Reply:
Yeah I just made it run every minute since it's not going to affect delay; let me fix the PR name